.circleci/community-apps-test.py

```diff
@@ -0,0 +1,92 @@
+import yaml
+import subprocess
+import sys
+from dotenv import set_key, load_dotenv
+
+def default_env(path, app_name):
+    with open(path, "w") as file:
+        file.write("")
+    set_key(path, 'HASURA_GRAPHQL_DATABASE_NAME', app_name, quote_mode="never")
+    set_key(path, 'HASURA_GRAPHQL_DATABASE_USER', 'gql_test', quote_mode="never")
+    set_key(path, 'HASURA_GRAPHQL_DATABASE_HOST', 'localhost', quote_mode="never")
+    set_key(path, 'HASURA_GRAPHQL_DATABASE_PORT', '5432', quote_mode="never")
+    set_key(path, 'HASURA_GRAPHQL_DATABASE_PASS', '', quote_mode="never")
+    set_key(path, 'HASURA_GRAPHQL_DATABASE_URL', 'postgres://gql_test@localhost:5432/{}'.format(app_name), quote_mode="never")
+    set_key(path, 'HASURA_GRAPHQL_SERVER_URL', 'http://localhost:8080', quote_mode="never")
+
+EXIT_FAILURE=False
+POSTGRES_DEFAULT_CMD=['psql', '-h', 'localhost', '-p', '5432', '-U', 'gql_test', '-c']
+DEFAULT_DOCKER_CMD=['docker', 'run', '--net', 'host']
+APPS_DATA = {}
+with open(".circleci/community-apps.yaml", 'r') as stream:
+    try:
+        APPS_DATA = yaml.safe_load(stream)
+    except yaml.YAMLError as exc:
+        print(exc)
+
+for app in APPS_DATA['apps']:
+    print("Running test for app {}".format(app['name']))
```

Comment:
Add a separator with some newlines and something like a long line of `------` at the beginning for each app?